src/gui/MiniMap.cpp

```diff
@@ -333,11 +333,11 @@ void MiniMap::showBookMiniMap(int showLevel) {
 		
 		if(showLevel == ARX_LEVELS_GetRealNum(m_currentLevel)) {
 			playerPos = computePlayerPos(zoom, showLevel);
-			start = Vec2f(490.f, 220.f) - playerPos;
+			start = Vec2f(rect.center()) - playerPos;
```

Comment:
Ok, the previous value might have been a bug then. Just wanted to make sure it wasn't an oversight.